cdap-app-templates/cdap-etl/cdap-etl-batch/src/main/java/co/cask/cdap/etl/batch/ETLBatchApplication.java

```diff
@@ -38,26 +48,33 @@
 public class ETLBatchApplication extends AbstractApplication<ETLBatchConfig> {
   public static final String SCHEDULE_NAME = "etlWorkflow";
   public static final String DEFAULT_DESCRIPTION = "Extract-Transform-Load (ETL) Batch Application";
+  private static final Set<String> SUPPORTED_PLUGIN_TYPES = ImmutableSet.of(
+    BatchSource.PLUGIN_TYPE, BatchSink.PLUGIN_TYPE, Transform.PLUGIN_TYPE,
+    Constants.CONNECTOR_TYPE, BatchAggregator.PLUGIN_TYPE);
```

Comment:
We should not have connector and batch aggregator for ETL Batch. It should be for data pipeline.